### PR TITLE
docs(cockpit): add traces documentation 

### PR DESCRIPTION
**Pull request by nerda-codes**
### Your checklist for this pull request

- Added concepts related to traces (Traces, TraceQL, Tempo)
- Added documentation on how to activate traces, how to retrieve traces push paths and how to push traces
- Added retention time info for traces in the Cockpit limitations page and FAQ
- Reworked other documentation to mention traces (token creation page, quickstart)

### CODE REVIEW

File: faq/cockpit.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ Cockpit integrates seamlessly with Scaleway’s ecosystem. It provides pre-built
 
 ## How am I billed for using Cockpit with my Scaleway data?
 
-Scaleway’s monitoring is included for free for a period of 30 days  for metrics and 7 days for logs.
+Scaleway’s monitoring is included for free for a period of 31 days for metrics and 7 days for logs.
```
Comment (bene2k1) [inline faq/cockpit.mdx:30]:
```suggestion
Scaleway’s monitoring is included for free for a period of 31 days for metrics and 7 days for logs.
```
Is it always 31 days even if a month has less than 31 days?

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Retrieve traces push paths
+
+<Message type="important">
+ Scaleway only supports the push paths mentioned below.
+</Message>
+
+Scaleway provides the following push paths for traces depending on the agent you are using:
+
+- The trace push path for the [Jaeger](https://www.jaegertracing.io/docs/1.50/) agent is `/jaeger`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud>/jaeger`.
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-push-traces.mdx:40]:
```suggestion
- The trace push path for the [Jaeger](https://www.jaegertracing.io/docs/1.50/) agent is `/jaeger`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud/jaeger>`.
```

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- The trace push path for the [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) agent is `/otlp`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud>/otlp`.
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-push-traces.mdx:41]:
```suggestion
- The trace push path for the [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) agent is `/otlp`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud/otlp>`.
```

Maybe remove the `< >` ?

File: observability/cockpit/how-to/activate-push-traces.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- The trace push path for the [Zipkin](https://zipkin.io/) agent is `/zipkin`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud>/zipkin`.
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-push-traces.mdx:42]:
```suggestion
- The trace push path for the [Zipkin](https://zipkin.io/) agent is `/zipkin`. The complete endpoint is `<https://traces.cockpit.fr-par.scw.cloud/zipkin>`.
```

File: faq/cockpit.mdx (hunk 1 above)
Comment (nerda-codes) [reply faq/cockpit.mdx:30]: It is actually 30 days, I'll fix it locally.

File: observability/cockpit/quickstart.mdx [hunk 5]
```diff
@@ -7,20 +7,21 @@ content:
...
+You can push logs with any Loki-compatible agent such as [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/), [Fluentd](https://docs.fluentd.org/), [Fluent Bit](https://docs.fluentbit.io/manual/) or [Logstash](https://www.elastic.co/guide/en/logstash/current/introduction.html).
+You can push traces with the Tempo-compatible agents supported by Scaleway: [Jaeger](https://www.jaegertracing.io/docs/1.50/), [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/), or [Zipkin](https://zipkin.io/).
 
 Read [our documentation](/observability/cockpit/reference-content/cockpit-limitations/) to find out which Scaleway products are integrated into Cockpit and learn [how to reduce extra costs and understand Cockpit pricing](/observability/cockpit/reference-content/understanding-cockpit-usage/).
 
-In this Quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials, how to create a push token and how to send metrics and logs to your Cockpit.
+In this Quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to create a push token to send metrics and logs to your Cockpit.
```
Comment (bene2k1) [inline observability/cockpit/quickstart.mdx:24]:
```suggestion
In this quickstart, we show you how to activate your Cockpit, how to retrieve your Grafana credentials and how to create a push token to send metrics and logs to your Cockpit.
```

**Review by bene2k1 — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**